shared/styles/mixins.scss

```diff
@@ -83,7 +97,7 @@
   padding: 0 $container-gutter-mobile;
   max-width: $page-width + $container-gutter-mobile * 1;
```

Comment:
this mobile stuff in `container` doesn't really do anything; `$page-width` is always going to be greater than mobile viewports anyway and I am not sure why there are differences between the base size (`$page-width` + ` $container-gutter-mobile * 1`) and min-mobile anyway? 

also, there may be instances where we need to override top & bottom padding / margins, so could left & right be explicitly declared as long-hand values?

Like: 

```
@mixin container() {
  margin-left: auto;
  margin-right: auto;
  padding-left: $container-gutter-mobile;
  padding-right: $container-gutter-mobile;

  @media (min-width: $min-tablet) {
    padding-left: $container-gutter-tablet;
    padding-right: $container-gutter-tablet;

    max-width: $page-width + $container-gutter-tablet * 2;
  }

  @media (min-width: $min-desktop) {
    padding-left: $container-gutter-desktop;
    padding-right: $container-gutter-desktop;

    max-width: $page-width + $container-gutter-desktop * 2;
  }
}

Reply:
It should definitely not use shorthand for padding.